psdk_wrapper/src/psdk_wrapper.cpp

```diff
@@ -412,6 +413,20 @@ PSDKWrapper::load_parameters()
                 "camera_frame param not defined, using default one: %s",
                 params_.camera_frame.c_str());
   }
+  if (!get_parameter("add_namespace_to_tf", params_.add_namespace_to_tf))
+  {
+    RCLCPP_WARN(get_logger(),
+                "add_namespace_to_tf param not defined, using default one: %s",
+                params_.add_namespace_to_tf ? "true" : "false");
+  }
+  if (params_.add_namespace_to_tf)
+  {
+    params_.imu_frame = std::string(get_namespace()) + "/" + params_.imu_frame;
```

Comment:
remove the "/" and just append the namespace. the separator can be in the frame id string